tools/castle-editor/code/framedesign.pas

```diff
@@ -2223,6 +2337,7 @@ procedure TDesignFrame.MenuItemViewportSort2DClick(Sender: TObject);
 
   ModifiedOutsideObjectInspector;
   UpdateDesign; // make the tree reflect new order
+  RecordUndo('MenuItemViewportSort2DClick'); //whatever that means?
```

Comment:
Should be `RecordUndo('Sort Items For Correct 2D Blending')`

Reply:
Thanks, fixed!